public/index.html

```diff
@@ -317,9 +316,7 @@ <h3 class="pl-1 fontHeading">Create New Task</h3>
             </div>
             <div class="form-group row">
                 <label for="description" class="col-sm-2 col-form-label">Description</label>
-                <div class="col-sm-10">
-                    <input type="text" class="form-control" id="desCreateNewTask">
-                </div>
+                <textarea type="text" class="col-sm-10" id="desCreateNewTask" rows="4"></textarea>
```

Comment:
1. change row number to 1.
2. add class="form-control" in textarea
3. give column in div and write textarea inside div...as it was written previously

Reply:
updated